### PR TITLE
chore: Enable explicit default timeout on all resources

### PR DESCRIPTION
**Pull request by remyleone**

### CODE REVIEW

File: scaleway/resource_baremetal_server.go [hunk 1]
```diff
@@ -22,8 +22,8 @@ func resourceScalewayBaremetalServer() *schema.Resource {
 		},
 		SchemaVersion: 0,
 		Timeouts: &schema.ResourceTimeout{
-			Create: &baremetalServerResourceTimeout,
-			Delete: &baremetalServerResourceTimeout,
+			Create: schema.DefaultTimeout(baremetalServerResourceTimeout),
```
Comment (jerome-quere) [inline scaleway/resource_baremetal_server.go:25]: Why not use default ?

File: scaleway/resource_instance_ip_reverse_dns.go [hunk 2]
```diff
@@ -18,6 +18,9 @@ func resourceScalewayInstanceIPReverseDNS() *schema.Resource {
 		Importer: &schema.ResourceImporter{
 			StateContext: schema.ImportStatePassthroughContext,
 		},
+		Timeouts: &schema.ResourceTimeout{
+			Default: schema.DefaultTimeout(instanceIPTimeout),
```
Comment (jerome-quere) [inline scaleway/resource_instance_ip_reverse_dns.go:22]: I wounder if we would not be easier to read if time was directly specified without using a varibale

**Review by jerome-quere — CHANGES_REQUESTED** [review]

File: scaleway/resource_instance_ip_reverse_dns.go (hunk 2 above)
Comment (remyleone) [reply scaleway/resource_instance_ip_reverse_dns.go:22]: I would prefer to have all timeout for a given set of resources be defined in a single place that allows for easy modification and reading once when we want to change the default for a given set of resources.

**Review by jerome-quere — CHANGES_REQUESTED** [review]
We should standardize the name of all timeout vars
```
default<ProductName><RessourceName>Timeout = XX;
```

**Review by jerome-quere — APPROVED** [review]

**Pull request merged**